Assets/Samples/2 - Microphone/MicrophoneDemo.cs

```diff
@@ -8,64 +8,65 @@ namespace Whisper.Samples
     public class MicrophoneDemo : MonoBehaviour
     {
         public WhisperManager whisper;
+        public MicrophoneRecord microphoneRecord;
         public bool streamSegments = true;
         public bool printLanguage = true;
 
-        [Header("Mic settings")] 
-        public int maxLengthSec = 30;
-        public int frequency = 16000;
-        public bool echo = true;
-
         [Header("UI")] 
         public Button button;
         public Text buttonText;
         public Text outputText;
         public Text timeText;
+        public Dropdown microphoneDropdown;
```

Comment:
I would recommend to make Dropdown optional field of `MicrophoneRecord`. This way it can be reused in other demos without boilerplate code from `MicrophoneDemo`